scanpy/tests/conftest.py

```diff
@@ -10,6 +10,7 @@
 
 import scanpy
 
+
```

Comment:
Is this a different rule than black?

Reply:
I used autopep8 to get rid of many violations. Could be that it introduced this line